restaurants_recommendation/users/views.py

```diff
@@ -57,3 +60,51 @@ def post(self, request: Request) -> Response:
         serializer.is_valid(raise_exception=True)
         serializer.save()
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class UserDetailView(APIView):
+    permission_classes = [permissions.IsAuthenticated]
+
+    @swagger_auto_schema(
+        operation_summary="유저 상세 정보 조회",
+        responses={status.HTTP_200_OK: UserSerializer},
+    )
+    def get(self, request: Request, user_id: int) -> Response:
+        """
+        사용자 상세 정보를 조회합니다.
+
+        Returns:
+            username (str): 사용자 계정 이름.
+            latitude (str): 사용자의 위치 중 위도
+            longitude (str): 사용자의 위치 중 경도
+            is_lunch_recommend (bool): 점심 메뉴 추천 사용여부
+            created_at (str): 사용자 계정 생성 일자
+            updated_at (str): 사용자 계정 수정 일자
+        """
+        user = get_object_or_404(User, id=user_id)
```

Comment:
```suggestion
        user = get_object_or_404(User, id=request.user.id)
```
위에서 말했던 request.user.id로 접근해보시는 것은 어떨까요?

Reply:
넵! authentication_classes = [JWTAuthentication]를 사용해서 코드 수정해보겠습니다!